vertica/datadog_checks/vertica/data/conf.yaml.example

```diff
@@ -33,6 +33,30 @@ instances:
     #
     port: 5433
 
+    ## @param include_tables - list of strings - required
+    ## The tables to include for metric collection.
+    #
+    include_tables:
+      - licenses
+      - license_audits
+      - system
+      - nodes
+      - projections
+      - projection_storage
+      - storage_containers
+      - host_resources
+      - query_metrics
+      - resource_pool_status
+      - disk_storage
+      - resource_usage
+      - version
+      - custom
+
+    ## @param exclude_tables - list of strings - optional
```

Comment:
Do we need to have an `exclude_tables` option? If customers just remove metric groups from the `include_tables`, the metrics will be skipped. This seems to be duplicate option and can be confusing. See the scylla integration option for an example of how we filter metric groups 
https://github.com/DataDog/integrations-core/blob/e3b96cd01d13fbd2b908e28ca0f022d0129be3dc/scylla/datadog_checks/scylla/data/conf.yaml.example#L53

Reply:
I've removed the `exclude_tables` option. Thanks!